src/Common/src/System/Net/NetworkInformation/UnixCommandLinePing.cs

```diff
@@ -88,7 +88,7 @@ public static long ParseRoundTripTime(string pingOutput)
             int msIndex = pingOutput.IndexOf("ms", afterTime);
             int numLength = msIndex - afterTime - 1;
             string timeSubstring = pingOutput.Substring(afterTime, numLength);
-            double parsedRtt = double.Parse(timeSubstring);
+            double parsedRtt = double.Parse(timeSubstring, System.Globalization.CultureInfo.InvariantCulture);
```

Comment:
Nit: could we add a using for `System.Globalization` instead of specifying the full name here?